### PR TITLE
Add debian distribution

### PR DESCRIPTION
**Pull request by darioalessandro**
adding debian distribution using `cargo-deb`

### CODE REVIEW

File: video-daemon/README.md [hunk 1]
```diff
@@ -20,3 +20,10 @@ RUST_LOG=info cargo run -- --user-id <user-id> --video-device-index 2 --meeting-
 
 URL can be your local webtransport server or prod https://transport.rustlemania.com
 ```
+
+# Compile deb
+
+1. Install `cargo-deb` with `cargo install cargo-deb`
+2. run `cargo deb` this  will generate the deb file at `target/debian/video-daemon...deb`
```
Comment (darioalessandro) [inline video-daemon/README.md:27]: @griffobeid adding deb distribution

File: .github/workflows/cut-video-daemon-dev.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  push:
+    branches:
+      - main
+      - add-debian-distribution
+    tags:
+      - '*'
+
+jobs:
+  build_and_package:
+    runs-on: ubuntu-latest
```
Comment (griffobeid) [inline .github/workflows/cut-video-daemon-dev.yaml:13]: Can we use Debian-latest?

**Review by griffobeid — APPROVED** [review]

File: .github/workflows/cut-video-daemon-dev.yaml (hunk 2 above)
Comment (darioalessandro) [reply .github/workflows/cut-video-daemon-dev.yaml:13]: heck yeah lets do that!

**Comment by darioalessandro** [comment]
Screenshot 2024-01-04 at 10 18 23 PM
@griffobeid  ci just hangs if I select debian-latest

**Pull request merged**